### ISSUE TITLE
Proposals for new entries

### ISSUE THREAD

**Issue by rodrigomelo9** [hdl/awesome#99]
Hi @umarcor I think that this issue can be used as a *Pinned issue*, to discuss to add new awesome projects.

To inaugurate it, what about:
* [x] [pp4fpgas (Parallel Programming for FPGAs)](https://github.com/KastnerRG/pp4fpgas) (a CC-BY-4.0 book about HLS).
* [x] [Mutation Cover with Yosys (MCY)](https://github.com/YosysHQ/mcy) (or maybe to be included as a reference for SBY?)
* [x] *LibreCores: el correo libre* as a monthly publication.
* [x] Also, I was thinking about RISC-V, as a whole (to avoid hundreds/thousands of entries ). I mean, to have the RISC-V entry, and a lot of references to related awesome projects.
- [ ] chips4makers (#173)

**Comment by umarcor** [comment]
I think that adding pp4fpgas is ok. MCY would fit as a reference inside Symbiyosys. LibreCores would fit as a reference inside Librecores. With regard to RISC-V, I'm not sure. I believe it should be split into multiple projects. The relevance is whether the project contains tools, cores, frameworks, etc. Not the specific architecture. That's a tag.

**Comment by rodrigomelo9** [comment]
- [x] [Princeton Reconfigurable Gate Array](https://github.com/PrincetonUniversity/prga) ([web](http://parallel.princeton.edu/prga/))

**Comment by JulianKemmerer** [comment]
Hello @umarcor requested that I propose adding PipelineC to hdl/awesome.

I wasnt sure if that meant a pull request or comment here. I prepared what I think was requested: a markdown with some yaml
[pipelinec.md](https://github.com/hdl/awesome/files/8110905/pipelinec.md)   (I didnt know what to put in the 'Talk' field?)

Thanks for the opportunity to share info about PipelineC!

Let me know if there is anything else you need

**Comment by umarcor** [comment]
> (I didnt know what to put in the 'Talk' field?)

It's the number of a "Discussion". I created https://github.com/hdl/awesome/discussions/228, so the number of the PipelineC "talk" is 228.

> I wasnt sure if that meant a pull request or comment here. I prepared what I think was requested: a markdown with some yaml

A PR would be good, but this is ok as well. The effort is actually filling the metadata (which you, as the author, know better). That was nicely detailed. Thanks!

- https://hdl.github.io/awesome/items/pipelinec/
- https://github.com/hdl/awesome/commit/d6bde3b1dda99ca8f6f849d1760507d308b6d77a

> Thanks for the opportunity to share info about PipelineC!

Thank you for building and sharing awesome projects!

> Let me know if there is anything else you need

Feel free to share other awesome references you know, or to improve the descriptions/metadata of the items.
We try not to add "all of our own stuff", but either add projects from others, or wait until someone else asks us to add ours.

**Comment by JulianKemmerer** [comment]
I would have definitely suggested Silice if it was not already there - and I poked the author of https://dfianthdl.github.io/ on Twitter just now - two of my favorite alternative HDLs - they have some fancier features than regular HDL, but aren't going all HLS on ya :+1:

**Comment by JulianDeepwave** [comment]
https://www.pygears.org/ is an interesting project for sure.

**Comment by umarcor** [comment]
@JulianKemmerer  indeed! Wanna propose a PR? See e.g. https://github.com/hdl/awesome/commit/d6bde3b1dda99ca8f6f849d1760507d308b6d77a

**Comment by JulianKemmerer** [comment]
> https://www.pygears.org/ is an interesting project for sure.

Apologies for the duplicate comment - problem having git accounts for work and personal on same pc.

I should have some time to put together a markdown file - might try to reach out to the author of the pygears proj first

**Comment by JulianKemmerer** [comment]
Hopefully ok if I give a basic markdown with some info:
[pygears.md](https://github.com/hdl/awesome/files/8128451/pygears.md)


 hoping to get some pygears folks eyes on this to adjust the verbiage/contribute

**Comment by StefanSredojevic** [comment]
@JulianKemmerer Thanks for sending me a message about this, I just made a pull request: https://github.com/hdl/awesome/pull/229

P.S I'm not a fully technical person so maybe I did something wrong here, let me know if I should change something :)
